python/jsbeautifier/__init__.py

```diff
@@ -117,7 +117,7 @@ def beautify_file(file_name, opts = default_options() ):
             stream = sys.stdin
             input_string = ''.join(stream.readlines())
         except Exception as ex:
-            print("Must pipe input or define at least one file.", file=sys.stderr)
+            print("Must pipe input or define input file.\n", file=sys.stderr)
```

Comment:
This text should be identical between the python and javascript implementations.  
See test https://github.com/bitwiseman/js-beautify/blob/master/js/test/shell-smoke-test.sh#L16 
How about the making the text be "Must pipe input or define at least one input file.\n" for both implementations?

Reply:
My concern was that the wording of "at least one file" indicates you can specify one or *more* files as argument for the tool.  However, the python CLI only accepts one.  Apparently, that's difference from the js CLI.

Anyway, I'm ok to revert this change if consistency with the js version is preferred, and also update PR #1317 to also use "at least one file".

Reply:
@thoger 
Or you could get the python version to take more than one file.  That would be awesome. 😄

Reply:
I reverted the wording change to unblock this PR.  A change to handle more than one input file is out of scope of this PR, IMO.